### DISCUSSION TITLE
Support config whitelist domain/IPs for our own project.

### DISCUSSION THREAD

**Discussion by leohxj** [nhost/nhost#1223] [Ideas]
Project Region/Subdomain is public when use JS-SDK, other people can access it in our website, it not safe.

Can we support some config that help us protect only used in our own domain/IPs, I know other BaaS has some solution:

- Firebase: [Firebase App Check](https://firebase.google.com/docs/app-check)
- AppWrite: can config domian/ips
- Supabase: suggest we config RLS (Row Level Security)

**Comment by elitan** [comment]
Just for us to understand, can you describe your use case a bit more? What's the real-world problem you're having and would like to solve by this suggestion?
